pages/button-group/permutations.page.tsx

```diff
@@ -0,0 +1,146 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import ScreenshotArea from '../utils/screenshot-area';
+import PermutationsView from '../utils/permutations-view';
+import createPermutations from '../utils/permutations';
+import { ButtonGroup, ButtonGroupProps } from '~components';
+
+const permutations = createPermutations<ButtonGroupProps>([
+  {
+    variant: ['icon'],
+    dropdownExpandToViewport: [true, false],
```

Comment:
This is not really needed because in permutation tests we do not run any interactions and the dropdown is only shown upon clicking. Besides, the it is the button dropdown responsible for the dropdown presentation and not button group.